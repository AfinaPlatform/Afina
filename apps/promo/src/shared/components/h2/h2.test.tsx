import H2 from "./h2";
import { render, screen } from "@testing-library/react";

describe("shared/h2", () => {
  it("render h2 html tag", () => {
    const { container } = render(<H2>Hello</H2>);
    expect(container.querySelector("h2")?.tagName).toBe("H2");
  });
  it("snapshot", () => {
    const { asFragment } = render(<H2>Hello</H2>);
    expect(asFragment()).toMatchSnapshot();
  });
});
