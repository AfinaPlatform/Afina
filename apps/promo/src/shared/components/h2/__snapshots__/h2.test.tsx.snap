// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shared/h2 snapshot 1`] = `
<DocumentFragment>
  <h2
    class="root"
  >
    Hello
  </h2>
</DocumentFragment>
`;
